app/src/main/java/com/morihacky/android/rxjava/RotationPersistWorkerFragment.java

```diff
@@ -66,6 +68,8 @@ public Integer call(Long aLong) {
         //_intsObservable = _intsObservable.share();
         storedIntsObservable = intsObservable.replay();
 
+        storedIntsSubscription = storedIntsObservable.connect();
```

Comment:
refcount and connect are similar, the main difference is that:
 - using refcount the observable start when on the first subscription and stop automatically when no observer are subscribed
 - using connect the observable start immediately and can be stopped using the subscription returned by the method

In this example I prefer to use connect method to manage manually the subscription and unsubscribe when the activity is destroyed (in onDestroy of worker fragment).
Replay method is useful to retrieve the data emitted when the activity is in background.
You can find more info on this topic in my book, but it's in Italian (I hope we'll translate it soon) and I don't think you can speak Italian :)
http://www.amazon.it/Android-Programmazione-avanzata-Fabio-Collini-ebook/dp/B00Z6XQSKW/